Documentation/btrfs-subvolume.rst

```diff
@@ -112,6 +112,10 @@ delete [options] [<subvolume> [<subvolume>...]], delete -i|--subvolid <subvolid>
         -i|--subvolid <subvolid>
                 subvolume id to be removed instead of the <path> that should point to the
                 filesystem with the subvolume
+
+        -R|--recursive
+                delete subvolumes beneath each subvolume recursively
+
```

Comment:
This should be very clear that it either requires CAP_SYS_ADMIN to delete all subvolumes or mount option user_subvol_rm_allowed or that it will not delete inaccessible subvoluems. Basically what doucmentation btrfs_util_delete_subvolume says in libbtrfsutil/README.md. Same for help text.